planner/core/hints.go

```diff
@@ -165,8 +168,34 @@ func (p *BlockHintProcessor) getCurrentStmtHints(hints []*ast.TableOptimizerHint
 	return p.QbHints[currentOffset]
 }
 
+func restoreOptimizerHint(hint *ast.TableOptimizerHint) string {
```

Comment:
It and `bindinfo.RestoreTableOptimizerHint` are the same.

Reply:
You are right. `restoreOptimizerHint` has been removed now, PTAL.